src/views/charts/PnlChart.tsx

```diff
@@ -91,50 +91,23 @@ export const PnlChart = ({
   const pnlData = useSelector(getSubaccountHistoricalPnl, shallowEqual);
   const subaccountId = useSelector(getSubaccountId, shallowEqual);
 
-  const [minimumRequestedZoomDomain, setMinimumRequestedZoomDomain] = useState(-Infinity);
+  const [periodOptions, setPeriodOptions] = useState<HistoricalPnlPeriods[]>([
+    HistoricalPnlPeriod.Period1d,
+  ]);
 
   const [selectedPeriod, setSelectedPeriod] = useState<HistoricalPnlPeriods>(
-    abacusStateManager.getHistoricalPnlPeriod() || HistoricalPnlPeriod.Period1d
+    HistoricalPnlPeriod.Period1d
   );
 
   /**
    * Default period in Abacus to 90d so that we can work with a larger dataset
    */
   useEffect(() => {
     abacusStateManager.setHistoricalPnlPeriod(HistoricalPnlPeriod.Period90d);
-  }, [pnlData]);
-
-  const onSelectPeriod = useCallback(
-    (periodName: string) => {
-      setSelectedPeriod(
-        HISTORICAL_PNL_PERIODS[
-          (periodName as keyof typeof HISTORICAL_PNL_PERIODS) || selectedPeriod.name
-        ]
-      );
-    },
-    [setSelectedPeriod, selectedPeriod]
-  );
+  }, []);
```

Comment:
🤔 would this be the same as using `HistoricalPnlPeriod.Period90d` on line 99?

Reply:
nope! the selectedPeriod state is the toggle value, the abacus state is how we determine what data to fetch; the chart is set up so that we only really need the 90d once and can ignore the other states

Reply:
i can update the default in abacus to 90d, but don't want to change too much from existing in case that affects mobile